### PR TITLE
Work Chat compatibility

### PR DESCRIPTION
**Pull request by chabou**
This PR adds compatibility with Work Chat, entreprise version of Messenger used with [Workplace](https://www.facebook.com/workplace).

3 parts:
* New menu items (and config) to switch to Work Chat and switch back to Messenger. When switching occurs, App is restarted.
* Starting URL and parsed URL depends on config item`useWorkChat`
* New CSS file to tweak Work Chat webapp.

Note: To log-out, menu creation after first simulated click is slower than Messenger. I had to put an ugly `setTimeout` to make it work.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ module.exports = new Store({
 			chatSeen: false,
 			typingIndicator: false
 		},
-		confirmImagePaste: true
+		confirmImagePaste: true,
+		useWorkchat: false
```
Comment (sindresorhus) [inline config.js]: Uppercase `c`

File: index.js [hunk 2]
```diff
@@ -133,6 +135,9 @@ function setUserLocale() {
 function createMainWindow() {
 	const lastWindowState = config.get('lastWindowState');
 	const isDarkMode = config.get('darkMode');
+	// Messenger or WorkChat
```
Comment (sindresorhus) [inline index.js]: Work Chat

**Comment by sindresorhus** [comment]
Can you add the feature to the readme? To https://github.com/sindresorhus/caprine#highlights and https://github.com/sindresorhus/caprine#features

**Comment by sindresorhus** [comment]
Would you be able to share some screenshots of how it looks in Work Chat mode? I don't have it.

Do you know anyone using Work Chat that could help test this?

**Comment by sindresorhus** [comment]
The login screen needs some styling tweaks, especially in dark mode.

- Remove the footer with the languages and fix the button for dark mode.
- Make the top part have transparent background. Add `background-color: transparent !important;` to `._5rw0`.
- Vertically center the login box.
- Make the window draggable by dragging the top of the window.

screen shot 2017-07-06 at 23 59 31

**Comment by chabou** [comment]
I'm using Work Chat @ work. Yes... I know... We don't use Slack... 🙄 
I (and few coworkers) can test it on Windows and Mac. My goal is to install it on my coworkers computer (~130 ppl) when released. 

Here's a screenshot. Quite the same than Messenger:
capture d ecran 2017-07-07 a 13 18 34

I can't reproduce your problem with your button (white background).
I noticed that it is not possible to drag window but this problem is on Messenger mode too. I thought it was ok. Maybe a separated PR to fix that is better?

I have to put a screenshot in README to explain how to Swtich to Work Chat.

All your other feedbacks are now fixed with af26c3cd731ff0a14b1d85ff0e87c4ca7fcfe9d1.

**Comment by sindresorhus** [comment]
> I can't reproduce your problem with your button (white background).

Press "Toggle Vibrancy" when in Dark Mode.

**Comment by sindresorhus** [comment]
> I have to put a screenshot in README to explain how to Swtich to Work Chat.

Seems your forgot to push that. https://github.com/sindresorhus/caprine/pull/245/files#diff-0730bb7c2e8f9ea2438b52e419dd86c9R104

**Comment by chabou** [comment]
> Seems your forgot to push that.

When I said "I have to put a screenshot in README", I meant: "I will put" :) It was not ready to merge, Sorry for the confusion. 

Problem with button reproduced, fix incoming.

**Comment by sindresorhus** [comment]
Thank you @chabou :) I'll do a new release in a couple of days. Just want to use this for a little bit to ensure it didn't introduce any regressions.

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/caprine/releases/tag/v2.5.0

**Pull request merged**